src/browser/components/MainPage.jsx

```diff
@@ -43,6 +43,9 @@ export default class MainPage extends React.Component {
       }
     }
 
+    // servers come from several places, in case we can't find the right one, pick the first one
+    key = key === -1 ? 0 : key;
```

Comment:
Just curiosity no changes required, @Willyfrog Normally which one seems more readable to you?

```
if(key === -1) {
  key = 0;
}
```
or the above conditional statement?

Reply:
For short comparisons the online one. For anything complex the if statement